pages/_document.js

```diff
@@ -0,0 +1,28 @@
+import Document, { Html, Head, Main, NextScript } from "next/document";
+
+class MyDocument extends Document {
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
+
+  render() {
+    return (
+      <Html>
+        <Head>
+          <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
does this really help? I would have thought link also causes prefetching?

Reply:
I'm not sure, this is what Google Font page suggested that I do.